### PR TITLE
Gsod 2023 ideas added

### PR DESCRIPTION
**Pull request by evasharma12**
**What kind of change does this PR introduce?**

Added new ideas for GSOD 2023 to the `docs/internships/gsod/ideas.md` page.

**Issue Number:**

#286 

**Summary**

- I removed the previous (Gsod 2022) ideas which got implemented last year.
- Added new ideas for Gsod 2023.
- Reviewed `docs/internships/ideas.md` page for any possible Gsod ideas.
- Kept one idea from Gsod 2022 which wasn't implemented last year.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: docs/internships/gsod/ideas.md [hunk 1]
```diff
@@ -107,32 +109,53 @@ There are others, but these are the main ones that will guide your contributions
...
+### Add Base Documentation to **Talawa** Repository
 
-Since the introduction of the CI/CD pipeline for automating documentation, there has not been much activity on the bulk of the older files, leaving most of the three codebases lacking in documentation. The goal of this task is to add comments on all of the existing methods, events and namespaces of all the classes within the Talawa, Talawa-Admin and Talawa API.
+Since the introduction of the CI/CD pipeline for automating documentation, there has not been much activity on the bulk of the older files, leaving most of the codebase lacking in documentation. The goal of this task is to add comments on all of the existing methods, events and namespaces of all the classes within the Talawa repository.
 
-* **Repos to update:** Talawa-API, Talawa, Talawa-Admin
-* **Skills Required:** The ideal candidate should be familiar with Javascript and in particular Node.js. Additionally, they need to be comfortable with using Git/Github and to update the documentation via pull requests.
+The previous Google Season of Docs (Gsod 2022) covered the rest two repositories: Talawa-admin, and Talawa-API.
```
Comment (tasneemkoushar) [inline docs/internships/gsod/ideas.md]: talawa-admin's documentation is not yet completed.

File: docs/internships/gsod/ideas.md [hunk 2]
```diff
@@ -107,32 +109,53 @@ There are others, but these are the main ones that will guide your contributions
...
+
+* **Repo to update:** Talawa
+* **Skills Required:** The ideal candidate should be familiar with Typescript and in particular Node.js. Additionally, they need to be comfortable with using Git/Github and to update the documentation via pull requests.
```
Comment (tasneemkoushar) [inline docs/internships/gsod/ideas.md]: In skills, do include familiarity with TSDoC.

File: docs/internships/gsod/ideas.md [hunk 3]
```diff
@@ -107,32 +109,53 @@ There are others, but these are the main ones that will guide your contributions
...
 * **How we measure sucess:** Increased documentation for files which have not been modified for more than 3 months. Additionally, an increase in the percentage of the documentation completed from the documentation coverage script that is used. 
-* **Possible Mentors:** Shannika Jackson, Dominic Mills, Michael Lue, Tasneem Koushar  
+* **Possible Mentors:** Eva Sharma
```
Comment (tasneemkoushar) [inline docs/internships/gsod/ideas.md]: Right now, don't include any names in the list of potential mentors. 
Once it gets decided, we will update that.

File: docs/internships/gsod/ideas.md [hunk 4]
```diff
@@ -142,15 +165,6 @@ The Talawa API has a number of GraphQL queries which are undocumented and have b
...
-
-* **Repos to update:** Talawa-API
-* **Skills Required:** HTML, Javascript, CSS. Any design skills would be a plus. Must also be comfortable using Github to submit changes.
-* **How we measure sucess:** A website structured in a completely different manner, which is more efficient and effective in providing the information for a user wishing contribute to Talawa. This can be measured by the increase in new contributors via pull requests.
-* **Possible Mentors:** Shannika Jackson, Dominic Mills, Michael Lue, Tasneem Koushar 
-* **Contact details:** Send your CV along with at least two technical writing samples to ![img](/img/email/mentors.png)
```
Comment (tasneemkoushar) [inline docs/internships/gsod/ideas.md:156]: could you add one more project section, for documentation of Talawa-admin.

Comment (evasharma12) [reply docs/internships/gsod/ideas.md:156]: Hey Tasneem,
I have added the additional features you asked for.
For adding another project section for documentation of Talawa-admin, I just mentioned it in the first project. Let me know if I should create another project.

**Review by tasneemkoushar — APPROVED** [review]

**Pull request merged**